src/constants/transformation/transform-service.ts

```
@@ -199,6 +199,69 @@ export class TransformService {
+            field.selectedValues.length > 0
+          ) {
+            const sv = field.selectedValues[0];
+            if (typeof sv === 'string') {
+              transformedData[camelKey] = sv;
+            } else if (typeof sv === 'object' && sv !== null) {
+              transformedData[camelKey] = sv.value ?? sv.id ?? null;
+            }
+            dynamicFieldLabels[camelKey] = label;
+          }
+
+        }
```

ESLint

[error] Delete `⏎`

(prettier/prettier)